### ISSUE TITLE
sha3 missing from install_requires (macOS10.12.3/python3.6)

### ISSUE THREAD

**Issue by wolfposd** [bigchaindb/bigchaindb#1128]
It appears pysha3 is currently not enough of a dependency

```bash
MAC:~ MAC$ bigchaindb
Traceback (most recent call last):
  File "/usr/local/bin/bigchaindb", line 7, in <module>
    from bigchaindb.commands.bigchain import main
  File "/usr/local/lib/python3.6/site-packages/bigchaindb/__init__.py", line 40, in <module>
    from bigchaindb.core import Bigchain  # noqa
  File "/usr/local/lib/python3.6/site-packages/bigchaindb/core.py", line 7, in <module>
    from bigchaindb.common import crypto, exceptions
  File "/usr/local/lib/python3.6/site-packages/bigchaindb/common/crypto.py", line 3, in <module>
    import sha3
  File "/usr/local/lib/python3.6/site-packages/pysha3-1.0.0-py3.6-macosx-10.12-x86_64.egg/sha3.py", line 8, in <module>
    from _sha3 import keccak_224, keccak_256, keccak_384, keccak_512
ImportError: cannot import name 'keccak_224'
```

however when installing sha3 (pip3 install sha3) the error disappears

**Comment by r-marques** [comment]
Wonder it this has to due with the fact that sha3 support for hashlib was added in python 3.6 https://docs.python.org/3/whatsnew/3.6.html#hashlib

**Comment by wolfposd** [comment]
seems that the error doesnt disappear when installing sha3, more errors appear when running.

```
MAC:~ MAC$ bigchaindb start
INFO:bigchaindb.commands.bigchain:BigchainDB Version 0.8.2
INFO:bigchaindb.config_utils:Configuration loaded from `/Volumes/HDD/MAC/.bigchaindb`
INFO:bigchaindb.db.utils:Create database `bigchain`.
INFO:bigchaindb.db.utils:Create `bigchain` table.
INFO:bigchaindb.db.utils:Create `backlog` table.
INFO:bigchaindb.db.utils:Create `votes` table.
INFO:bigchaindb.db.utils:Create `bigchain` secondary index.
INFO:bigchaindb.db.utils:Create `backlog` secondary index.
INFO:bigchaindb.db.utils:Create `votes` secondary index.
INFO:bigchaindb.db.utils:Create genesis block.
Traceback (most recent call last):
  File "/usr/local/bin/bigchaindb", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/site-packages/bigchaindb/commands/bigchain.py", line 329, in main
    utils.start(create_parser(), sys.argv[1:], globals())
  File "/usr/local/lib/python3.6/site-packages/bigchaindb/commands/utils.py", line 95, in start
    return func(args)
  File "/usr/local/lib/python3.6/site-packages/bigchaindb/commands/bigchain.py", line 179, in run_start
    db.init()
  File "/usr/local/lib/python3.6/site-packages/bigchaindb/db/utils.py", line 180, in init
    b.create_genesis_block()
  File "/usr/local/lib/python3.6/site-packages/bigchaindb/core.py", line 598, in create_genesis_block
    block = self.prepare_genesis_block()
  File "/usr/local/lib/python3.6/site-packages/bigchaindb/core.py", line 576, in prepare_genesis_block
    transaction = transaction.sign([self.me_private])
  File "/usr/local/lib/python3.6/site-packages/bigchaindb/common/transaction.py", line 943, in sign
    tx_partial_dict = tx_partial.to_dict()
  File "/usr/local/lib/python3.6/site-packages/bigchaindb/common/transaction.py", line 1197, in to_dict
    tx_id = Transaction._to_hash(tx_serialized)
  File "/usr/local/lib/python3.6/site-packages/bigchaindb/common/transaction.py", line 1227, in _to_hash
    return hash_data(value)
  File "/usr/local/lib/python3.6/site-packages/bigchaindb/common/crypto.py", line 9, in hash_data
    return sha3.sha3_256(data.encode()).hexdigest()
  File "/usr/local/lib/python3.6/site-packages/sha3/__init__.py", line 97, in sha3_256
    return SHA3256(s)
  File "/usr/local/lib/python3.6/site-packages/sha3/__init__.py", line 30, in __init__
    self._s = _sha3.sha3()
AttributeError: module '_sha3' has no attribute 'sha3'
```

**Comment by sohkai** [comment]
I'm not quite sure what the issue is, since everything looks fine on pysha3's side, but I [filed an issue there](https://github.com/tiran/pysha3/issues/4) to clarify.

At this point my best guess is that the keccak implementations aren't getting built in the distributions for 3.6.

----------

BTW, you won't get very far running BigchainDB natively on OSX/macOS (due to some python multiprocessing primitives not being available). We'd recommend using [docker](https://docs.bigchaindb.com/projects/server/en/latest/appendices/run-with-docker.html) (or [`docker-machine` with `docker-compose`](https://github.com/bigchaindb/bigchaindb/blob/master/docker-compose.yml)) instead :).

**Comment by sohkai** [comment]
This should be fixed with `pysha3@v1.0.2`.
